README.md

```diff
@@ -67,4 +67,65 @@ Any of the 3 category identifiers can be used: Category_Id, Category_Code, Edit_
         "canonical_uri": "/category-uno-buno"
     }
 }
-```
\ No newline at end of file
+```
+
+### WishListItemAndOptions_InsertOrUpdate
+
+This function allows you to add a product to a customer's wishlist
+
+**Request Body Parameters**
+
+| Name | Type | Description |
+| --- | --- | --- |
+| Module_Function | String | WishListItemAndOptions_InsertOrUpdate |
+| Customer_Login | String | Customer Login to use. |
+| Product_Code | String | Product code to use. |
+| Quantity | Number | Quantity of product you wish to add. |
+| Wishlist_ID | Number | _Optional_ Wishlist ID. |
+| Attributes | Array | Optional, unless the product requires attributes. |
+
+```
+Attributes is an array with the following parameters for each attribute:
```

Comment:
This should say something like Attributes is an array of objects with the following properties for each attribute object:

Reply:
Should also specify the Name, Type, and Description for the Attribute object.

Reply:
No you

Reply:
Are we able to add fields for wish list notes, date/time an item was added, and/or if the list is public or private?

Reply:
Are you talking about the Wishlist or the items
Date/time no, that is hardcoded in miva.
Wishlist Item notes, yes i can make a spot for that.

For wishlist, not the items, that would need to be a seperate function

Reply:
wishlist - title / public vs private
wishlist item - notes

Reply:
Now that the function is changing, i will need to change the docs. So i'm going to ignore that for now and make the updates when necessary.